samples/iam.js

```diff
@@ -111,7 +187,7 @@ async function removeBucketIamMember(bucketName, roleName, members) {
   const bucket = storage.bucket(bucketName);
 
   // Gets and updates the bucket's IAM policy
-  const [policy] = await bucket.iam.getPolicy();
+  const [policy] = await bucket.iam.getPolicy({requestedPolicyVersion: 3});
```

Comment:
Add conditional policy check to make sure a condition isn't set when removing a member from a role. It explicitly tells a reader that the code ignores bindings with conditions.

Reply:
👍 fixed!